src/api/users/user.controller.ts

```diff
@@ -34,9 +36,10 @@ export class UserContoller{
     @ApiResponse({ status: 400, description: 'Invalid password'})
     @ApiResponse({ status: 404, description: 'User not found'})
     @UsePipes(new ValidationPipe())
-    async loginUser(@Body() loginUserDto: LoginUserDto): Promise<UserResponseInterface> {
+    async loginUser(@Body() loginUserDto: LoginUserDto, @Res() response: Response): Promise<Response> {
         const user = await this.userService.loginUser(loginUserDto);
-        return this.userService.buildUserResponse(user);
+        response.set({ 'authorization':  this.userService.generateJwt(user)});
+        return response.status(201).json({ message: "User authorized" });
```

Comment:
Same as above and status should be 200 since this endpoint isn't supposed to create new data.